.github/workflows/release.yml

```diff
@@ -163,10 +237,11 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
           check-latest: true
 
-      - name: Create manifests
+      - name: Create KLT manifest
+        if: needs.release-please.outputs.klt-release-created == 'true'
         env:
           RELEASE_REGISTRY: ghcr.io/keptn
-          CHART_APPVERSION: ${{ needs.release-please.outputs.tag_name }}
+          CHART_APPVERSION: ${{ needs.release-please.outputs.klt-tag-name }}
```

Comment:
this is tricky... I believe for the moment using the KLT tag is fine, but we might reconsider this in future

Reply:
This is fine for now but will be revisited in #1600